concourse/pipelines/create-bosh-cloudfoundry.yml

```diff
@@ -1867,7 +1867,7 @@ jobs:
                 - -e
                 - -c
                 - |
-                  if [ -n "${TF_VAR_datadog_api_key}" -a -n "${TF_VAR_datadog_app_key}" -a "${ENABLE_DATADOG}" = "false" ]; then
+                  if [ -n "${TF_VAR_datadog_api_key}" ] && [ -n "${TF_VAR_datadog_app_key}" ] && [ "${ENABLE_DATADOG}" = "false" ]; then
```

Comment:
According to shellcheck the exact behaviour of `-a` and `-n` are not well specified between implementations, so `&&` and `||` are preferred.